index.bs

```diff
@@ -300,15 +300,20 @@ parallel</a>:
 
 <pre class="idl">
 partial interface DocumentOrShadowRoot {
-  readonly attribute HTMLVideoElement? pictureInPictureElement;
+  readonly attribute Element? pictureInPictureElement;
 };
 </pre>
 
 The {{pictureInPictureElement}} attribute's getter must run these steps:
 
 1. If the <a>context object</a> is a <a for=/>shadow root</a> and its
```

Comment:
step 1's wording be redundant.

You can say simply:
- If the context object is not *connected*, return null and abort these steps.

Reply:
Done. Thank you @hayatoito!